LetsDoStuff.Domain/stylecop.json

```diff
@@ -0,0 +1,14 @@
+﻿{
```

Comment:
Have you added the file intentionally? If so, please, remove the automatic comments

Reply:
I added it accidentally and I even hadn`t noticed it until I saw your comment. 
Should I remove it?

Reply:
sure